modules/oadp-upgrading-oadp-operator-1-2-0.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * backup_and_restore/oadp-release-notes.adoc
+
+:_mod-docs-content-type: PROCEDURE
+
+[id="upgrading-dpa-operator-1-2-0_{context}"]
+= Upgrading the OADP Operator
+
+Use the following sequence when upgrading the {oadp-first} Operator.
+
+.Procedure
+. Change your subscription channel for the OADP Operator from `stable-1.1` to `stable-1.2`.
+. Allow time for the Operator and containers to update and restart.
+
+For general Operator upgrade instructions, see xref:../../operators/admin/olm-upgrading-operators.adoc#olm-changing-update-channel_olm-upgrading-operators[Updating installed Operators].
```

Comment:
You can't have an xref in a module. This needs to be in an `.Additional resources` section in the assembly.

Reply:
I moved it to the assembly.